community/2023-05-10-community-meeting.md

```diff
@@ -17,6 +17,113 @@ import ReactPlayer from 'react-player/youtube';
+  - You could run two KVs but there's no way to know which one to talk to by default, which feels artificially restrictive
+  - No concrete suggestions, but we've got some people with hands
+- Jordan: Concrete suggestion, rename the project to Wild Wild wasmCloud west.
+  - More seriously: compiling link names to actors -- it's been a bit of a pain-point
+- Kevin: COmment on Brooks' usecase (implicit links)
+  - In addition to the security problem, you could have many different providers all on the same link ID on the same lattice, and they could pick the wrong one (ex. looking for redis, and you find vault)
+  - Brooks: definitely still things to think about here, but this settles it for me (for now)
+- Brooks: What should we take away from this discussion? What about Jordan's question from Slack about compiling link names
+  - Jordan: Should washboard have a section that says "these builtins are up and running"
+  - Jordan: Other question was answered -- Kevin says it's in the works
+  - Bailey: we should probably write some RFCs
+- Jordan: If we do a bunch of hosts with different providers and builtins and lattice them all together, how do we...
+  - Kevin: if the list of builtins is variable and we don't have contracts that specify that they're builtins
+  - If an actor comes up on a host has a builtin for wasi:logging and there is no link definition, then the host *should* hook the actor up to the builtin wasi logging
+    - Only if no link exists for the contract would the builtin be used
+    - If you deploy a distributed logger, then a linkdef should be added to stay meaningful, explicit and durable
+    - We do want a hierarchy of defaults here
+  - Brooks: Love that, we should totally get an RFC
+- Kevin: one example of reasonable defaults -- right now you have to deploy an HTTP provider to give client requests... But a host could easily provide this, and then you don't need a link definition
+  - An issue is that if you have multiple different hosts -- one with a default and another one without -- then you could get one when you didn't expect it
+
+#### wasmCloud: Path to a CNCF Incubating project
+
+- Bailey: This will be quick because most of the criteria are actually already hit!
+  - The journey starts at the incubating stage -- and one of the things they ask you to do is to look at the graduated stuff and see if you're
+  - Right now we're at the sandbox stage
+  - What we expect to be at soon (tm) is the incubating stage
+- Bailey: Call for adopters -- please let us know how you're using wasmcloud
+- See the [OpenSSF best practices (via badge on wasmCloud repo)](https://bestpractices.coreinfrastructure.org/en/projects/6363)
+- Brooks: I'm a wasmcloud user, how do I get in contact you
+  - Bailey: If you're comfortable with it post it in the wasmCloud Slack! If not, send a direct message to me on Slack!
```

Comment:
```suggestion
  - Bailey: If you're comfortable with it post it in the [wasmCloud Slack](https://slack.wasmcloud.com)! If not, send a direct message to me on Slack!
```

Reply:
great minds think alike!  I already changed this at the same time I tihnk